### ISSUE TITLE
Preview Tab

### ISSUE THREAD

**Issue by MichaelGHUser** [bluesky-social/social-app#1215]
![image](https://github.com/bluesky-social/social-app/assets/123146201/e24d08fd-4062-4f1f-a994-884eb7192c92)

**Is your feature request related to a problem? Please describe.**

Since posts cannot be edited, a Preview tab would be a nice addition. 

**Describe the solution you'd like**

A preview would be great to catch typos, odd formatting, or dead links before posting.

**Additional context**

Additionally, this would be a useful Accessibility option: a _'Require preview before posting'_ toggle, if you will.

**Comment by intrnl** [comment]
I'm not sure what the benefit of this would be given that in Bluesky, you're writing plain text whereas in something like GitHub where you can attach inline images and videos, and code snippets

**Comment by MichaelGHUser** [comment]
You can attach images to Bluesky posts.  Video support is coming at a later date, afaik.

**Comment by intrnl** [comment]
Yeah, but they're not inline like GitHub, you can see images without the need for a separate preview tab
